components/mixins/experience-event/experienceevent-advertising-impression-details.schema.json

```diff
@@ -0,0 +1,44 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/experienceevent-advertising-impression-details",

```

Comment:
Would you consider adding these properties with in another object (e.g "impressionDetails") under the existing "experienceevent-advertising" mixin  ?